src/storage/exec/FilterNode.h

```diff
@@ -68,7 +68,7 @@ class FilterNode : public IterateNode<T> {
             auto result = filterExp_->eval(*expCtx_);
             // NULL is always false
             auto ret = result.toBool();
-            if (ret.second && ret.first) {
```

Comment:
Because the where/when conditions of this go/update/upsert are all handled here

If you only support the go where condition, the field must be of type bool or string.
Fields that do not support the when condition of update/upsert must be of bool type or string type. The update/upsert when condition will be filtered out here instead of a syntax error.

so, please handle the when and where conditions of go. update upsert uniformly.

Reply:
> Because the `where/when` conditions of this `go/update/upsert` are all handled here
> 
> If you only support the `go where` condition, the field must be of type` bool` or `string`.
> Fields that must be of bool type or string type do not handle the `when `condition of `update/upsert` .
> The `update/upsert` `when `condition will be filtered out here instead of a syntax error.
> Causes an error in the result of the `update/upsert` statement
> 
> so, please handle the `when and where` conditions of `go update upsert` uniformly.
> @laura-ding @czpmango

ACK.

Reply:
Fixed in [https://github.com/vesoft-inc/nebula-graph/pull/772](https://github.com/vesoft-inc/nebula-graph/pull/772) 
and [https://github.com/vesoft-inc/nebula-graph/pull/767](https://github.com/vesoft-inc/nebula-graph/pull/767)